--- Python/dynload_win.c	Sat Oct 12 14:43:13 2013 +0800
+++ Python/dynload_win.c	Sat Oct 12 16:23:18 2013 +0800
@@ -25,6 +25,7 @@
 };
 
 
+#ifndef __MINGW32__
 /* Case insensitive string compare, to avoid any dependencies on particular
    C RTL implementations */
 
@@ -41,6 +42,7 @@
 
     return (first - second);
 }
+#endif /* __MINGW32__ */
 
 
 /* Function to return the name of the "python" DLL that the supplied module
